share/spack/gitlab/cloud_pipelines/stacks/aws-pcluster-neoverse_v1/packages.yaml

```diff
@@ -0,0 +1,63 @@
+---  # Neoverse N1 / V1 packages
+packages:
+  acfl:
+    require:
+      - one_of: ["%gcc target=aarch64"]
+        message: "Clang based compilers need GCC libraries and they should be made available for the wide range of CPUs they actually support.
+                Edit $SPACK_ROOT/etc/spack/packages.yaml to change this default."
+  gromacs:
+    require:
+      - one_of:
+          - "gromacs@2021.3 %arm ^fftw ^openmpi"
+          - "gromacs@2021.3 %gcc ^armpl-gcc ^openmpi"
+  libfabric:
+    buildable: false
+    externals:
+      - prefix: /opt/amazon/efa/
+        spec: libfabric@${LIBFABRIC_VERSION} fabrics=efa
+  llvm:
+    variants: ~lldb
+  mpas-model:
+    require: "precision=single make_target=llvm %arm ^parallelio+pnetcdf"
+  mpich:
+    require: "mpich pmi=pmi2 device=ch4 netmod=ofi +slurm"
+  nvhpc:
+    require:
+      - one_of:
+          - "nvhpc %gcc target=aarch64"
+        message: "NVHPC should be built with GCC and should be made available for the wide range of CPUs they actually support.
+                Edit $SPACK_ROOT/etc/spack/packages.yaml to change this default."
+  openfoam:
+    require: "openfoam %gcc ^scotch@6.0.9"
+  openmpi:
+    variants: ~atomics ~cuda ~cxx ~cxx_exceptions ~internal-hwloc ~java +legacylaunchers ~lustre ~memchecker +pmi +romio ~singularity +vt +wrapper-rpath fabrics=ofi schedulers=slurm
+    require: '@4:'
+  # Spack cannot read cflags="", re-enable when https://github.com/spack/spack/issues/39671 is fixed
```

Comment:
This is resolved now (https://github.com/spack/spack/issues/39671)

Reply:
Yes, thank you. Unfortunately I ran into another (issue)[https://github.com/awslabs/palace/pull/207] in building palace. I have updated the comment.